src/NSwag.CodeGeneration.CSharp/Templates/File.liquid

```diff
@@ -8,10 +8,14 @@
 using {{ usage }};
 {% endfor -%}
 
+#pragma warning disable  108 // Disable "CS0108 '{derivedDto}.ToJson()' hides inherited member '{dtoBase}.ToJson()'. Use the new keyword if hiding was intended."
```

Comment:
Can't we just use `#pragma warning disable` as it was before?

Reply:
Using `pragma warning disable all` would mask any new warnings that my changes, or other people's changes, might introduce. Personally I would fix the code to address all of the warnings instead of hiding them, but if we're going to hide them then we should be explicit about it.

Reply:
So if we remove the ExposeJsonSerializerSettings and never expose or assume it is only changed for initialization we could reuse the serializer?

Reply:
(I think you meant to post this in reply to a different comment) Yes. But that would be a breaking change. I'll leave that up to you.

Reply:
Ok. Fine for me. I hope you catched all warnings in all possible generation paths :-)